src/vs/workbench/browser/parts/editor/breadcrumbsModel.ts

```diff
@@ -250,7 +261,16 @@ export class EditorBreadcrumbsModel {
 	private _isFiltered(element: TreeElement): boolean {
 		if (element instanceof OutlineElement) {
 			const key = `breadcrumbs.${OutlineFilter.kindToConfigName[element.symbol.kind]}`;
-			return !this._configurationService.getValue<boolean>(key);
+
+			// If possible, look for language specific overrides when reading configuration.
+			if (this._editor && this._editor.getModel()) {
+				const model = this._editor.getModel() as ITextModel;
+				return !this._configurationService.getValue<boolean>(key, {
+					overrideIdentifier: model.getLanguageIdentifier().language
+				});
+			}
+
+			return this._configurationService.getValue<boolean>(key);
```

Comment:
You can use the [ITextResourceConfigurationService](https://github.com/microsoft/vscode/blob/9727eb60500657cd2c30e4a3992e17940c2cefba/src/vs/workbench/api/common/extHostTypeConverters.ts#L243) which makes this a lot simpler

Reply:
I tried this implementation:
```typescript
if (this._editor && this._editor.getModel()) {
	const model = this._editor.getModel() as ITextModel;
	return this._textResourceConfigurationService.getValue<boolean>(model.uri, key);
}
```

It works great, except for the boolean value being flipped! The values are always opposite of what's expected.

This is true for both language-specific overrides and the base config.

Reply:
How's this? 9d9a99d67562315c34c195865c0720abff96c721